program/src/state.rs

```diff
@@ -721,6 +724,10 @@ impl Validator {
             msg!("Refusing to remove validator because it still has stake accounts, unstake them first.");
             return Err(LidoError::ValidatorShouldHaveNoStakeAccounts);
         }
+        if self.has_unstake_accounts() {
+            msg!("Refusing to remove validator because it still has unstake accounts, withdraw them first.");
+            return Err(LidoError::ValidatorShouldHaveNoUnStakeAccounts);
```

Comment:
Nit: We call the instruction `Unstake`, but here it’s written `UnStake` with capital S.